src/dlgTriggerEditor.cpp

```diff
@@ -8370,15 +8365,50 @@ void dlgTriggerEditor::resizeEvent(QResizeEvent* event)
 void dlgTriggerEditor::slot_key_grab()
 {
     mIsGrabKey = true;
+    setShortcuts(true);
+    QCoreApplication::instance()->installEventFilter(this);
+}
+
+// Activate shortcuts for editor menu items like Ctrl+S for "Save Item" etc.
+// Deactivate instead with optional "true" - useful to grab for a keybinding
+void dlgTriggerEditor::setShortcuts(const bool unsetInstead)
```

Comment:
Can the logic be switched so that providing a `false` value means "unset"? It is not not a good idea to avoid the sort of implied double-negative that this currently suggests... don't you not agree? :stuck_out_tongue_winking_eye:

Reply:
See below